src/GitLink/Program.cs

```diff
@@ -8,6 +8,9 @@
 namespace GitLink
 {
     using System;
+    using System.CommandLine;
```

Comment:
Right. It embed all its own dependencies so it's not an issue here. Otherwise I totally agree with you.